lib/rom/sql/extensions/postgres/commands.rb

```diff
@@ -12,22 +12,26 @@ module Create
           def insert(tuples)
             tuples.map do |tuple|
               relation.dataset.returning(*relation.columns).insert(tuple)
-            end.flatten
+            end.flatten.map { |t| relation.output_schema[t] }
           end
 
           # Executes multi_insert statement and returns inserted tuples
           #
           # @api private
           def multi_insert(tuples)
-            relation.dataset.returning(*relation.columns).multi_insert(tuples)
+            relation.dataset.returning(*relation.columns).multi_insert(tuples).map { |t|
+              relation.output_schema[t]
+            }
```

Comment:
wow you already updated this to use `output_schema` OOOK :D

anyhow, I *think* it would be better to use an after hook for that, something like this:

``` ruby
module ROM
  module SQL
    class Create < ROM::Commands::Create
      after :finalize

      def finalize(tuples, *)
        with_input_tuples(tuples).map { |t| relation.output_schema[t] }
      end
    end
  end
end
```

Should be more flexible like that. WDYT?